README.md

```diff
@@ -108,6 +123,7 @@ The project has been developed using Eclipse, and can be easily imported in such
   * If you are a Windows user: `gradlew.bat alchemist-incarnation-biochemistry:generateGrammarSource`
 
   Go to Eclipse, right click on `alchemist-incarnation-biochemistry` project -> Refresh
+* If you have errors in `alchemist-projectview` project, make sure Xtend plugin is correctly installed
```

Comment:
I already explained it in the "Recommended configuration" part (rows 72-74 of the updated README.md).
I have to copy/move that after that "troubleshooting" part (row 126)?